ecchronos-binary/src/pylib/ecchronoslib/table_formatter.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 #
 
+from __future__ import print_function
```

Comment:
is this needed?

Reply:
Yes, this enforces that we use print as a function rather than a statement:
```
print "hello"
vs
print("hello")
```

Also, without it pylint breaks in python 2.7:
```
C: 39, 0: Unnecessary parens after 'print' keyword (superfluous-parens)
C: 40, 0: Unnecessary parens after 'print' keyword (superfluous-parens)
C: 41, 0: Unnecessary parens after 'print' keyword (superfluous-parens)
C: 47, 0: Unnecessary parens after 'print' keyword (superfluous-parens)
C: 48, 0: Unnecessary parens after 'print' keyword (superfluous-parens)

```

In 3.x of python is not a statement anymore.